### ISSUE TITLE
Typo: DOMAINE

### ISSUE THREAD

**Issue by Firsh** [thibaut-d/docker-traefik-wordpress#1]
In docker-compose.yml https://github.com/thibaut-d/docker-traefik-wordpress/blob/master/site/docker-compose.yml in the labels you have a typo because of which it shows an error in traefik.

`      - "traefik.http.routers.wordpressmysite.rule=Host(`${DOMAINE}`)"`

**Comment by DaggieBlanqx** [comment]
Yes, there is a typo in **line 56** or ``site/docker-compose.yml``.
It is supposed to be `` - "traefik.http.routers.wordpressmysite-secured.rule=Host(`${DOMAIN}`)"`` not `` - "traefik.http.routers.wordpressmysite-secured.rule=Host(`${DOMAINE}`)"``

I have submitted a PR #4

**Comment by Anvnamnet** [comment]
2 year later.. Boy it do be hard to fix a typo ;)
